src/components/filter/FilterBoilerplate.vue

```diff
@@ -229,6 +229,7 @@ export default {
   },
   data() {
     return {
+      // TODO: hasValues is a computed property, don't know how to fix this one
       collapseItems: this.collapsedIfNoValues && !this.hasValues,
```

Comment:
`hasValues` here is a computed property, I tried to move the whole `collapseItems` to ba a computer property but it breaks the implem. Do you have a better suggestion?

Reply:
It seems that this is something coming from vue 1.x (https://v2.vuejs.org/v2/guide/migration.html#cache-false-deprecated). The migration guide recommands to transform it as a method and call it like a method
```
     L9: 'filter--has-values': hasValues()
     L233: collapseItems: this.collapsedIfNoValues && !this.hasValues(),
```

from:
```
computed:{
     hasValues: {
        cache: false,
        get() {
          return this.$store.getters['search/hasFilterValues'](this.filter.name)
        }
    }
}
```
to
```
methods:{
     hasValues(): {
          return this.$store.getters['search/hasFilterValues'](this.filter.name)
    }
}
```